Add Juanita's introductory blogpost

juanis2112 opened the pull request:
- I added my introductory blogpost 
- I added my information to the authors.yaml and my headshot
- I added the images for the blogpost

I thought we could take a picture together this weekend @lwasser and I could add it to the 'Looking ahead' section but apart from that I think it is ready. I also built the website to check everything was correct but I don't have the pre-commit hooks installed yet so that might fail. I'm happy to install them and run them if you can guide me through it.

I would appreciate feedback on the figure captions and ALT text since that was not part of the initial draft and also please let me know if you have some specific guidelines for image compression or if the files are too big.

lwasser commented on _posts/2023-05-20-introduction-juanita.md:
```diff
@@ -0,0 +1,94 @@
+layout: single
+title: "🎶 My Journey Through Scientific Python: The Lyrics 🎶
+ "
+excerpt: "Hi there, I'm Juanita. I recently joined pyOpenSci as a community lead to help with community building and outreach and this blogpost tells the story of how I got here. "
```
This is what search engines use to post information about this post. so i suggest a bit more straight forward excerpt that helps a reader know what this content is about!
```suggestion
excerpt: "Juanita Gomez recently joined pyOpenSci as a community lead to help with community building and outreach. Learn more about Juanita's path to becoming a leader in community management in the scientific Python ecosystem. "
```

lwasser commented on _posts/2023-05-20-introduction-juanita.md:
```diff
@@ -0,0 +1,94 @@
+title: "🎶 My Journey Through Scientific Python: The Lyrics 🎶
+ "
+excerpt: "Hi there, I'm Juanita. I recently joined pyOpenSci as a community lead to help with community building and outreach and this blogpost tells the story of how I got here. "
+author: "Juanita Gomez"
+author: "Juanita Gomez"
+permalink: /blog/journey-through-scientific-python.html
```
```suggestion
author: "Juanita Gomez"
permalink: /blog/journey-through-scientific-python.html
```

lwasser commented on _posts/2023-05-20-introduction-juanita.md:
```diff
@@ -0,0 +1,94 @@
+
+## Introduction: About Me
+
+Hello, everyone! I'm Juanita, and I'm excited to share my story with you. I'm currently a computer science PhD student at UC Santa Cruz and the community manager of the Scientific Python project (which you will hear more about in a bit 👀). I’m originally from Colombia, and I completed my undergraduate studies in pure mathematics at Universidad Javeriana while also taking some music courses. I'm a ...
```
let's add a link to the scientific python home page here!

lwasser commented on _posts/2023-05-20-introduction-juanita.md:
```diff
@@ -0,0 +1,94 @@
+
+<figure>
+    <a href="/images/introduction-juanita/juanita-guitar.png">
+    <img src="/images/introduction-juanita/juanita-guitar.png" style="max-width:100%" alt="A woman sitting on top of a hill in the dessert holding a guitar.">
```
```suggestion
    <img src="/images/introduction-juanita/juanita-guitar.png" style="max-width:100%" alt="A woman sitting on top of a hill in the desert holding a guitar.">
```

lwasser reviewed (changes requested):
This is so great @juanis2112 thank you!! i made some comments about making the headers a bit more descriptive for people who skim text.

I also think we should pull out a few quotes in the blog and highlight them.

i did not build this locally yet. maybe you can make the changes i requested and then i can suggest where the quotes could go once we have a final draft. 

wonderful job on this!!

CAM-Gerlach commented on _posts/2023-05-20-introduction-juanita.md:
```diff
@@ -0,0 +1,94 @@
+
+## Verse 1: The Spark
+
+One of my earliest contributions to the open-source world was with the [Spyder IDE](https://www.spyder-ide.org) when I worked as a Software Engineer at Quansight. I primarily focused on writing documentation and engaging in community outreach. My initial task was to update the documentation for Spyder 4, the latest release at that time, and make it more user-friendly.
```
If you need it... [SVG](https://github.com/spyder-ide/spyder/blob/master/img_src/spyder.svg). [PNG](https://github.com/spyder-ide/spyder/blob/master/img_src/spyder.png)

CAM-Gerlach commented on _posts/2023-05-20-introduction-juanita.md:
```diff
@@ -0,0 +1,94 @@
+
+## Verse 1: The Spark
+
+One of my earliest contributions to the open-source world was with the [Spyder IDE](https://www.spyder-ide.org) when I worked as a Software Engineer at Quansight. I primarily focused on writing documentation and engaging in community outreach. My initial task was to update the documentation for Spyder 4, the latest release at that time, and make it more user-friendly.
```
Our standard short-form descriptor for Spyder is "the scientific Python development environment", since it pretty succinctly (and hopefully clearly) describes what it is/what it does, and is the origin of the name. So you could say:

```suggestion
One of my earliest contributions to the open-source world was with [Spyder](https://www.spyder-ide.org), the scientific Python development environment, when I worked as a Software Engineer at Quansight. I primarily focused on writing documentation and engaging in community outreach. My initial task was to update the documentation for Spyder 4, the latest release at that time, and make it more user-friendly.
```

If you want a screenshot, you could use [this one](https://github.com/spyder-ide/spyder/blob/master/img_src/screenshot.png), the (relatively) up to date hero on our web page and docs that Juanita herself took (her username is actually visible in the file paths).

juanis2112 commented:
@lwasser

-  I fixed the images sizes and alignments. I'm pretty happy with how It looks now.
- I removed the highlight from the oldest post
- I added our picture to the pyOpenSci section 
- I added the peer review link to the pyOpenSci section 
- I  haven't changed the titles of the sections since I'm not convinced about this yet. I didn't want the section titles to be that long and I also like that some of them are more metaphorical so not sure how to be more descriptive keeping this idea. Do you have any thoughts on this? I thought about keeping the titles and adding an extra part, eg. The Spark: Spyder IDE but since at the beginning each section already has the name of the part of the song, it would be kind of weird. Unless I remove it which I think I would not want. Or unless it's something like 'Verse 1: The Spark - Spyder IDE'. What do you think?
I'm leaning towards having them a little bit more abstract and short since I think it goes better with the whole 'mood' of the text but I'm happy to hear more suggestions on how to solve this issue.

lwasser commented:
ok wonderful!! i'm going to test our our new dev container that @crazy4pi314 made for us to see how this looks without building locally :)

lwasser commented on _posts/2023-05-20-introduction-juanita.md:
```diff
@@ -0,0 +1,103 @@
+
+## Outro: Gratitude
+
+As a community manager at both Scientific Python and pyOpenSci, I have witnessed the wonders of the Scientific Python ecosystem. I have encountered countless remarkable individuals and witnessed the incredible projects they have created using the available tools. In my opinion, unifying the community and promoting project integration is the key to unlocking the full potential of this ecosystem. I ...
```
@juanis2112 this is really good. i wanted to clarify the position because again we have a full time community manager coming on board so i don't want any confusion as she has been hired. So i just adjusting the language here and also made scientific Python lowercase to avoid confusion (which already exists!) because pyOpenSci, the scientific Python broader community and the Scientific Python project. If you are ok with this we can merge this and then merge the entire blog. just let me know.

CAM-Gerlach commented:
Seems like you can ignore the spurious htmlproofer linkcheck failure on Linkdin by adding the URL [under `--ignore-urls` in the workflow file](https://github.com/juanis2112/pyopensci.github.io/blob/intro-post/.github/workflows/build-site.yml#L41). Otherwise, it will presumably fail on every subsequent push and PR (the other spurious internal link failure should correct itself once this is merged).

> ok wonderful!! i'm going to test our our new dev container that @crazy4pi314 made for us to see how this looks without building locally :)

Have you considered just using something like Netlify for automatic site previews on PRs? The free tier should be more than enough and it only takes a [few simple steps to set up](https://www.netlify.com/blog/2020/04/02/a-step-by-step-guide-jekyll-4.0-on-netlify/#connecting-to-netlify). We use that or something similar on all the sites I'm involved with; we even used it for the Python docs until we switched to RTD for previews. Once we started using it, it was so handy to have an instant, fully working preview on every PR (both for reviewers and especially contributors) that it feels like going back to the stone age without it, haha.

lwasser commented on _data/authors.yml:
```diff
@@ -40,3 +40,17 @@ Anita Graser:
       url: "https://twitter.com/underdarkGIS"
+Juanita Gomez:
+  name        : "Juanita Gomez"
+  bio         : "Community Lead, pyOpenSci & Scientific Python"
```
```suggestion
  bio         : "Community Lead, Scientific Python"
```
I checked in on this position. Essentially because we are hiring a full position in this space and this position ends June 15 which is next week i think we should keep the Scientific Python position but remove pyOpenSci so I don't get in legal trouble with hiring and pay and such.

lwasser commented:
@CAM-Gerlach i have thought about implementing that. i just haven't had time!!  i did set it up for another website i'm working on. it was easy. i wasn't sure if the previews would be free forever if we update the site regularly?? maybe we can open an issue and discuss?

CAM-Gerlach commented:
Sure. Opened as #197

lwasser commented on _posts/2023-05-20-introduction-juanita.md:
```diff
@@ -0,0 +1,103 @@
+    <a href="/images/introduction-juanita/scientific-python.png">
+    <img src="/images/introduction-juanita/scientific-python.png" style="max-width:100%" alt="A group of people standing in front of the AT&T Hotel and Conference Center at the University of Texas campus in Austin, Texas wearing blue and black shirts with the Scientific Python logo. ">
+    </a>
+    <figcaption>This is a picture of the people who attended our <a href="https://www.youtube.com/watch?v=R9hNnhpMgTE&t=2s"> Scientific Python community outreach BOF session </a> at SciPy 2022. We gave everyone who attended Scientific Python shirts! It was an amazing discussion with people from a lot of different projects within the Scientific Python ecosystem.
```
```suggestion
    <figcaption>This is a picture of the people who attended the <a href="https://www.youtube.com/watch?v=R9hNnhpMgTE&t=2s"> Scientific Python community outreach BOF session </a> at SciPy 2022. We gave everyone who attended Scientific Python shirts! It was an amazing discussion with people from a lot of different projects within the Scientific Python ecosystem.
```

lwasser commented:
Good morning @juanis2112 . I wanted to leave a bit of feedback that I have after talking this blog through with my board a bit the other day. One of the community challenges that we have at pyOpenSci is that it's confusing for people regarding what pyOpenSci is and what we are doing vs what the Scientific Python project is doing. I know that Stefan and Jarrod get similar questions! 

In the blog post verse 2 is about your amazing work with the Scientific Python ecosystem. And the focus they have is really development work around the bigger core packages which is super needed and critically important!  
<img width="774" alt="Screen Shot 2023-06-14 at 8 49 39 AM" src="https://github.com/pyOpenSci/pyopensci.github.io/assets/7649194/794be8f5-97d4-44d8-beba-0c08aff06358">

The following paragraph - the chorus is also about talks you've given for Scientific Python. Which again is wonderful but it's really focused on that specific effort!  
<img width="818" alt="Screen Shot 2023-06-14 at 8 51 00 AM" src="https://github.com/pyOpenSci/pyopensci.github.io/assets/7649194/95423458-0e74-4a50-910b-600d8826f48b">

There is a small section on pyOpenSci and supporting our work as well. But this internship that you have been doing with us is wrapping up this week. it's been so so wonderful to work with you!!! What a gift! 

Then the final paragraph is about Scientific Python project again with an image of everyone in that project with the tshirts. I thijnk what happens is the broader scientific Python ecosystem (people who are scientists using Python vs the Scientific Python project)  is conflated in this blog. 
<img width="856" alt="Screen Shot 2023-06-14 at 9 01 49 AM" src="https://github.com/pyOpenSci/pyopensci.github.io/assets/7649194/f39c38c2-39ce-429d-afb7-e945bff025d2">

The efforts of Scientific Python project and pyOpenSci are super complementary! But the communities and target audiences are actually quite different. I worry this blog post mixes the two together in a way that will confuse folks more. And this has been a real challenge for me when i got into communities to talk. 

But it also **really** speaks to your devoted support for and engagement with the Scientific Python project (which i love!). Your incredible work with them SHOULD be highlighted and spotlighted.  I just wonder if this post should live on the Scientific Python blog rather than ours given it's really focused on that community? 

i am concerned about this blog post confusing folks more. Let's talk this through a bit together this morning and see where we land. I just wanted to voice my concerns here openly so I can be transparent.

lwasser commented:
thank you, @willingc . i'll close it now!!
